cpu/nrf52/radio/nrf802154/nrf802154_radio.c

```diff
@@ -253,7 +253,7 @@ static int _confirm_op(ieee802154_dev_t *dev, ieee802154_hal_op_t op, void *ctx)
 {
     (void) dev;
     bool eagain;
-    ieee802154_tx_info_t *info = NULL;
+    ieee802154_tx_info_t *info = (ieee802154_tx_info_t *)ctx;
```

Comment:
```suggestion
    ieee802154_tx_info_t *info = ctx;
```

no need to cast the `void *` - squash directly

Reply:
Thanks